### PR TITLE
Pulumi Insights page updates for reinvent 2024

### PR DESCRIPTION
**Pull request by shughes26**
### Proposed changes
Insights layout updates
- ~needs update graphics~ graphics are updated
- lots of lorem ipsum / ESC content pasted in

### CODE REVIEW

File: content/product/pulumi-insights.md [hunk 1]
```diff
@@ -8,43 +8,65 @@ aliases:
...
 
-        - title: Improve Operations
-          icon: shield
+        - title: Efficiency
+          icon: cycle
           icon_color: blue
           description: |
             Analyze and update infrastructure to optimize costs, enhance security, and ensure compliance. 
 
+workflow:
```
Comment (aaronkao) [inline content/product/pulumi-insights.md:36]: I think there needs to be some text right under Workflow. Should say something like

"Making sense of the millions of cloud resources and resource updates per month, across hundreds of clouds, regions, and accounts is hard. Pulumi Insights provides you with the tools to to navigate all four phases of infrastructure management"

File: layouts/product/pulumi-insights.html [hunk 2]
```diff
@@ -64,7 +97,7 @@ <h4 class="p-0 m-0 text-gray-800">{{ .subtitle }}</h4>
         </div>
     </section>
 
-    <section id="analytics" class="container my-28 mx-auto text-center px-6">
+    <section id="analytics" class="container my-20 mx-auto text-center px-6">
```
Comment (aaronkao) [inline layouts/product/pulumi-insights.html:100]: I don't think we need this section. I think we can delete it. I do like the shadowed box it is using. Can we drop the pricing section into a similar box?

**Review by aaronkao — COMMENTED** [review]
Some changes

File: content/product/pulumi-insights.md [hunk 3]
```diff
@@ -8,43 +8,65 @@ aliases:
...
-    titleTop: Intelligent Cloud Management
-    titleBottom: 
+    header: Intelligent Cloud Management
+    title: A central hub to securely manage all of your environments, secrets, and configurations
+    body: |
+      - **Stop secret sprawl.** Pull and sync secrets and configuration with any secrets store – HashiCorp Vault, AWS Secrets Manager, Azure Key Vault, GCP Secret Manager, 1Password, and more – and consume in any application, tool, or CI/CD platform.  
+      - **Trust (and prove) your secrets are secure.** Adopt dynamic, short-lived secrets on demand as a best practice. Lock down every environment with RBAC, versioning, and a full audit log of all changes.  
+      - **Ditch `.env` files.** No more copying-and-pasting secrets or storing them in plaintext on dev computers. Developers can easily access secrets via CLI, API, Kubernetes operator, the Pulumi Cloud UI, and SDKs.
+      - **Use with or without Pulumi IaC.** Use Pulumi ESC independently, or use with Pulumi IaC to support storing secrets in config in a more secure way than using plaintext.
     description: |
```
[the remark this answers is not in the document]
Comment (shughes26) [reply content/product/pulumi-insights.md:15]: It was deleted in html but I missed deleting the field from markdown, will fix it

File: layouts/product/pulumi-insights.html [hunk 4]
```diff
@@ -64,48 +98,19 @@ <h4 class="p-0 m-0 text-gray-800">{{ .subtitle }}</h4>
...
-                    <img class="mx-auto lg:mr-0 rounded shadow-lg" src="{{ .image }}" alt="Pulumi Insights AI" />
+                <p class="mx-auto">{{ .description | markdownify }}</p>
+                <div class="mt-12">
+                    <a href="/pricing" class="btn-secondary">View Pricing</a>
                 </div>
-            {{ end }}
-        </section>
-    </div>
-
-    <section id="pricing" class="container mx-auto text-center mt-28 px-6">
```
Comment (aaronkao) [inline layouts/product/pulumi-insights.html:98]: I think pricing box needs to be the same size as the Get Started purple box below

**Comment by aaronkao** [comment]
As part of this PR, can you also remove the waitlist please? Redirect it to this page.

**Comment by aaronkao** [comment]
@shughes26 I think its the bottom images of this [Figma](https://www.figma.com/design/51Zdaexm7YjODmN5e5SDJa/Pulumi-Website-Diagrams?node-id=250-2&node-type=canvas&t=VALRH2HoQOGqYXcY-0). Can you update all the images with these new properly sized images? Thanks

File: content/product/pulumi-insights.md [hunk 5]
```diff
@@ -6,30 +6,58 @@ meta_desc: Pulumi Insights is Intelligent Cloud Management. Gain security, compl
...
+  - header: Resource Search
+    subheader: Find anything in any cloud
+    body: Ask any questions about your infrastructure across more than 100 clouds, using either structured search queries or natural language prompts. Search helps you find that needle in the haystack – locating a single resource across many clouds and environments – as well as running sophisticated queries such as tracking down untagged or expensive resources across the whole organization.
+    graphic: /images/product/insights-resource-search.png
+  - header: Policy Violations
+    subheader: Discover violations and enforce compliance
+    body: Pulumi Insights provides a comprehensive view of all policy violations across your organization, helping you maintain visibility and control over your infrastructure.
+    graphic: /images/product/insights-policy.png
+  - header: AI Copilot
+    subheader: Have a conversation about your infrastructure
```
Comment (aaronkao) [inline content/product/pulumi-insights.md:58]:
```suggestion
    subheader: Converse about your infrastructure
```

**Comment by aaronkao** [comment]
@shughes26 Can you update the video to this link: https://youtu.be/hXXRVbURKsQ

Then can you update the video thumbnail too please. Thanks
insights-video-thumbnail

**Comment by thtmnisamnstr** [comment]
Add a new sub-section to the Key Features section. Copy below (Aaron will reply with the image).
Headline:  Scan & Sync AWS Resources
Sub-headline:  Gain Visibility Across All Your AWS Resources
Body:  Pulumi Insights scans and syncs your entire AWS infrastructure, including resources not managed Pulumi IaC. This brings all the features of Pulumi Insights to all your organization’s cloud infrastructure.

File: content/product/pulumi-insights.md [hunk 6]
```diff
@@ -6,30 +6,58 @@ meta_desc: Pulumi Insights is Intelligent Cloud Management. Gain security, compl
...
 - /insights
+- /product/waitlist/
 
 overview:
-    titleTop: Intelligent Cloud Management
-    titleBottom: 
-    description: |
-        Pulumi Insights is an asset and compliance management service that gives search, compliance remediation, resource visualizations, and AI insights over any infrastructure, including resources not provisioned by Pulumi IaC such as AWS CloudFormation, Microsoft ARM, HashiCorp Terraform, or even manually.
+    header: Intelligent Cloud Management
+    title: Asset management, compliance remediation, resource visualizations, and AI insights over the cloud
```
Comment (aaronkao) [inline content/product/pulumi-insights.md:13]:
```suggestion
    title: Asset management, compliance remediation, and AI insights over the cloud
```
Can we make it always 2 lines?

**Review by thtmnisamnstr — APPROVED** [review]

**Pull request merged**